README.md

```diff
@@ -158,17 +158,13 @@ helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
       cd -
       ```
   - [Option 2] Namespace scoped deployment
-    - Specify the namespace to listen to
```

Comment:
add a liner about which namespace it will listen on. For its sample it will be what ACK_K8S_NAMESPACE is set to